### DISCUSSION TITLE
Question about binning

### DISCUSSION THREAD

**Discussion by tbilloud** [villekf/OMEGA#28] [Q&A] [answered]
Hi,

if I change the binning in the gate_CT_main.m example (e.g. 2 instead of 1) I get the attached error. Am I missing something? I didn't change anything else in the script (the x/y sizes are divided and the pixel pitch multiplied by the binning).

Cheers,
Thomas

[Screen Shot 2021-09-20 at 1.38.04 PM.pdf](https://github.com/villekf/OMEGA/files/7195367/Screen.Shot.2021-09-20.at.1.38.04.PM.pdf)

**Comment by villekf** [comment, marked as answer]
Hi,

There was a bug in the code that caused data load to fail when using binning greater than 1. You can download the fixed function from https://raw.githubusercontent.com/villekf/OMEGA/master/source/loadProjectionData.m and replace the old one in the OMEGA source-folder. That should fix the issue.

**Reply by tbilloud** [reply]
thanks !
